apps/loader/chunked_upload.js

```diff
@@ -400,3 +400,46 @@ function continueGoogleDriveUpload(token) {
         });
   }, 3000);
 }
+
+// to upload a file from Dropbox
+function dropboxChooserStart(){
+  //refer docs: https://www.dropbox.com/developers/chooser
+  options = {
+
+    // Required. Called when a user selects an item in the Chooser.
+    success: function(files) {
+        handleUpload(files);
+    },
+
+    // Called when the user closes the dialog without selecting a file
+    // and does not include any parameters.
+    cancel: function() {
+
+    },
+
+    // Optional. "preview" (default) is a preview link to the document for sharing,
+    // "direct" is an expiring link to download the contents of the file.
+    linkType: "direct", // or "direct"
+
+    // Optional. A value of false (default) limits selection to a single file, while
+    // true enables multiple file selection.
+    multiselect: true, // or false
```

Comment:
I'm not sure if you should leave this as true.
Since multi file upload is not allowed from the normal upload flow. We have batch loader for that.
@birm What do you think?

Reply:
Yes, makes sense. Thanks for pointing it out!

Reply:
*disable multiselect in the following commit